### ISSUE TITLE
Multi-project Board

### ISSUE THREAD

**Issue by 99brgs** [kanboard/forum#105]
Hello
I have several projects that are working together and I would like to view the tasks from all of them in one screen. Is this possible? Thanks in advance

**Comment by creecros** [comment]
One board, or just any old screen?

**Comment by 99brgs** [comment]
Yes. It would be useful to have the chance of seeing the status of the whole work in one board, showing the project name. In my case, all the projects have the same swimlanes but if not, all the different swimlanes should be shown.

**Comment by creecros** [comment]
No, not out the box. You could view all the tasks in multiple projects in a search.

**Comment by 99brgs** [comment]
Thanks

**Comment by creecros** [comment]
https://github.com/stinnux/kanboard-bigboard

That's what you are looking for I believe.
